daml-assistant/src/DAML/Assistant/Types.hs

```diff
@@ -82,7 +84,8 @@ data VersionOptions = VersionOptions
 -- | Command-line options for daml install command.
 data InstallOptions = InstallOptions
     { iTargetM :: Maybe RawInstallTarget -- ^ version to install
-    , iActivate :: ActivateInstall -- ^ activate the assistant
+    , iAssistant :: InstallAssistant -- ^ install the assistant
+    , iActivate :: ActivateInstall -- ^ install the assistant if true (deprecated, delete with 0.14.x)
```

Comment:
I’d probably handled the logic for `--activate` in the parser and then only have `iAssistant` here but I don’t feel strongly about this.

I’m also not entirely sure we need backwards compatibility here but the added complexity seems small enough that I’m definitely not opposed to having it.

Reply:
I think I still need it here for the `--activate` deprecation warning. 

Yeah ... getting rid of `iActivate` and `--activate` is definitely something I want to do, perhaps in `0.14.x`. I just think that now that we have users we need to be careful not break user interfaces too quickly.